src/main/java/com/conveyal/gtfs/GTFSCache.java

```diff
@@ -248,6 +274,7 @@ private GTFSFeed retrieveFeed (String originalId) {
                 throw new RuntimeException(e);
             }
         } else {
+            LOG.warn("Feed {} found neither locally nor on S3", originalId);
```

Comment:
Is it reached though if it's not on S3? Won't we have already thrown an exception?